### ISSUE TITLE
Document prerequisites

### ISSUE THREAD

**Issue by oleks** [NixOS/nix#930]
On a clean Raspbian Jessie Lite (2016-05-27) I needed to do this:

```
sudo apt-get install libssl-dev libbz2-dev libsqlite3-dev libcurl3-openssl-dev liblzma-dev
export SQLITE3=sqlite3
cpan DBI
cpan DBD::SQLite
cpan WWW::Curl
```

It is a bit depressing that the seemingly best way to spot these requirements is to rerun `./configure` until it succeeds, or fails with one of the above requirements missing. A list of requirements in the `INSTALL` file would've been nice. I can make a pull request for such a list myself, unless you have objections, or better thoughts on how to "fix" this issue.

**Comment by oleks** [comment]
Of course, I now found the [prerequisites in the Nix manual](https://nixos.org/nix/manual/#sec-prerequisites-source), but it would've be nice to link to this page in the `INSTALL` file, if not list them again in the `INSTALL` file.

**Comment by b-m-f** [comment]
@thufschmitt There does not seem to be an `INSTALL` file, that is mentioned in the OP.
Since you reopened this issue I assume you might have some insights. Could you give me some information on how to work on this?

**Comment by thufschmitt** [comment]
@b-m-f I guess you want https://nixos.org/manual/nix/stable/installation/prerequisites-source ?
It would be nice to have it in that repo

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
